make/stratos/run

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+. "${BASH_SOURCE%/*}/_env"
+. "${BASH_SOURCE%/*}/_has_release.sh"
+
+# TODO(f0rmiga): Refactor make/run and this file to move the following variables to a single source.
+: "${DOMAIN:=cf-dev.io}"
+: "${UAA_HOST:=uaa.${DOMAIN}}"
+: "${UAA_PORT:=2793}"
+: "${UAA_CLIENT_ID:=cf}"
+: "${UAA_CLIENT_SECRET:=""}"
+: "${UAA_ADMIN_IDENTIFIER:=cloud_controller.admin}"
+: "${UAA_SKIP_SSL_VALIDATION:=true}"
+: "${STORAGE_CLASS:=$(kubectl get storageclass 2>/dev/null | awk '/(default)/ { print $1 ; exit }')}"
+
+tmp_stratos="/tmp/stratos"
+mkdir -p $tmp_stratos
+stratos_installer="${tmp_stratos}/installer.tgz"
+stratos_chart="${tmp_stratos}/console/"
+download_log="${tmp_stratos}/download.log"
+printf "Downloading Stratos release to ${tmp_stratos}... "
+curl -L -o $stratos_installer $STRATOS_URL 2> $download_log
```

Comment:
Please run `shellcheck` on scripts (in this case, it would flag not quoting the variable).